lib/logflare_web/live/endpoints/endpoints_live.ex

```diff
@@ -0,0 +1,213 @@
+defmodule LogflareWeb.EndpointsLive do
```

Comment:
This one might need some love afterwards to extract some stuff to functions and cleanup but later 👀